Spell out "anisotropic" fully in filter options

skyace65 opened the pull request:
For some reason this was shortened to "aniso" which seems like it could cause confusion, especially since searching "aniso" online doesn't show any results for "anisotropic." Also the filter setting in base material fully spells out anisotropic. Either the description for this needs to be changed or the description for base material needs to be changed.

skyace65 committed:
Spell out "anisotropic" fully in filter options

akien-mga commented on scene/main/canvas_item.cpp:
```diff
@@ -986,7 +986,7 @@ void CanvasItem::_bind_methods() {
 
 	ADD_GROUP("Texture", "texture_");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "texture_filter", PROPERTY_HINT_ENUM, "Inherit,Nearest,Linear,Nearest Mipmap,Linear Mipmap,Nearest Mipmap Aniso.,Linear Mipmap Aniso."), "set_texture_filter", "get_texture_filter");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "texture_filter", PROPERTY_HINT_ENUM, "Inherit,Nearest,Linear,Nearest Mipmap,Linear Mipmap,Nearest Mipmap Anisotropic.,Linear Mipmap Anisotropic."), "set_texture_filter", "get_texture_filter");
```
The dots should be removed too.

skyace65 commented:
I have removed the dots.

Zireael07 commented:
Typo in PR title - I hope the commit itself doesn't have it?

akien-mga commented:
There's another occurrence of the same in that file:
```
$ rg "Aniso[^t]"
scene/main/canvas_item.cpp
963:    ADD_PROPERTY(PropertyInfo(Variant::INT, "texture_filter", PROPERTY_HINT_ENUM, "Inherit,Nearest,Linear,Nearest Mipmap,Linear Mipmap,Nearest Mipmap Aniso.,Linear Mipmap Aniso."), "set_texture_filter", "get_texture_filter");
1303:   ADD_PROPERTY(PropertyInfo(Variant::INT, "texture_filter", PROPERTY_HINT_ENUM, "Inherit,Nearest,Linear,Nearest Mipmap,Linear Mipmap,Nearest Mipmap Aniso.,Linear Mipmap Aniso."), "set_texture_filter", "get_texture_filter");
```

There are more occurrences of "aniso" or "ANISO" in the code that might warrant refactoring but this could be done later.

Calinou commented:
> For some reason this was shortened to "aniso" which seems like it could cause confusion

I shortened it in the PR that renamed the property hints, as it made the dropdown very long otherwise. It's not unusable by any means, but I thought it looked strange.

skyace65 commented:
Fixed the typo and fixed the second occurrence. I checked and I don't think it looks any weirder long compared to short, in both cases the dropdown shifts to the right
![NothingWeird](https://user-images.githubusercontent.com/6080368/186283091-24625a9d-2967-4cc7-bd18-a15d37507f4f.PNG)
![AlsoNothingWeird](https://user-images.githubusercontent.com/6080368/186283096-a47136ef-9bf3-45ba-a006-d19c7d158164.PNG)
.

akien-mga commented:
Thanks!
